.github/workflows/release.yml

```
@@ -57,11 +51,45 @@ jobs:
         name: enarx-${{ matrix.platform.target }}-oci
         path: enarx-${{ matrix.platform.target }}-oci
 
+  aarch64-build:
```

```suggestion
  build-aarch64:
```

And original `build` should be renamed to `build-x86_64`